views/index.pug

```diff
@@ -0,0 +1,84 @@
+html
```

Comment:
Your example routes are all wrong. They claim `/cookies/something` but your index.js is mapped with `/api/something`